KStudy.js

```diff
@@ -0,0 +1,173 @@
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.indexOf('/journal/thesis_name.asp')>-1) {
+		return "journalArticle";
+	} else if ((url.indexOf('/journal/list_name.asp')>-1 || url.indexOf('/search/result_kiss.asp')>-1) && getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//li[@class="title"]/a');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		if (href.indexOf("javascript")>-1) {
+			var tnameAndKey = href.match(/detailView\(\'([\S]+)\'\,\'([\S]+)\'/);
+			var journalhref = "../journal/thesis_name.asp?tname=" + tnameAndKey[1] + "&key=" + tnameAndKey[2];
+			href = journalhref;
+		}
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	var item = new Zotero.Item("journalArticle");
+	item.title = ZU.xpathText(doc, '//div[@class="agency_box"]/div[1]/span');
+	item.language = "kr";
+	
+	var creators = ZU.xpathText(doc, '//div[@class="agency_detail"]/ul/li[1]');
+	if (creators) {
+		var creatorsList = creators.split(',');
+		for (var i=0; i<creatorsList.length; i++) {
+			item.creators.push(ZU.cleanAuthor(creatorsList[i], "author", true));
+		}
+	}
+	
+	var details = ZU.xpathText(doc, '//div[@class="agency_detail"]/ul/li[2]');
+	//e.g. details = "한국공자학회, <공자학> 27권0호 (2014), pp.179-210"
+	//Z.debug(details);
+	if (details) {
+		item.date = ZU.strToISO(details);
+		var publicationTitle = details.match(/<([^>]+)>/);
+		if (publicationTitle) {
+			item.publicationTitle = publicationTitle[1];
+		}
+		var pagerange = details.match(/p+\.(\d+-\d+)/);
+		if (pagerange) {
+			item.pages = pagerange[1];
+		}
+		var volumeIssue = details.match(/>\s+(\d+)\D(\d+)/);
+		if (volumeIssue) {
+			item.volume = volumeIssue[1].replace(/^0/, '');
+			item.issue = volumeIssue[2].replace(/^0/, '');
+		}
+	}
+	
+	item.abstractNote = ZU.xpathText(doc, '//div[@class="con_detail"]/div[4]/ul');
+	if (item.abstractNote) {
+		item.abstractNote = ZU.trimInternal(item.abstractNote);
+	}
+
+	var pdfJSurl = ZU.xpathText(doc, '//div[@class="search_box"]/div[@class="choice"]/span[1]/a/@href');
+  	var pdfurlKeys = pdfJSurl.match(/Select1\(\'(\S+)\'\,\s?\'(\S+)\'\,\s?(\S+)\,\s?(\S+)\,\s?(\S+)\)\;/);
+  	var pdfUrl = pdfurlKeys[1].replace(/^(.*\/).*/, "$1") + "viewer.asp?code_num=" + pdfurlKeys[2];
+
+	item.attachments.push({
+		url : pdfUrl,
+		title : "KISS PDF",
```

Comment:
let's make that `KStudy Full Text PDF`, please (we want "full text" in there for uniformity and the actual name of the translator, too (though there's some disagreement on that).